src/services/orderbook/redux/selectors.ts

```diff
@@ -14,5 +15,16 @@ export function selectMyOrders(state: IAppReduxState) {
 }
 
 export function selectOrders(state: IAppReduxState) {
-  return selectState(state).data.orders;
+  const orders = selectState(state).data.orders;
+  const hideOrders = selectHideOrders(state);
+  const filteredOrders = {...orders, records: getFilteredOrders(orders.records, hideOrders)};
+  return filteredOrders;
+}
+
+export function selectHideOrders(state: IAppReduxState) {
+  return selectState(state).data.hideOrders;
+}
+
+function getFilteredOrders(orders: IOrderList['records'], hideOrders: NS.TokenId[]) {
+  return orders.filter(order => !hideOrders.includes(order.tokenId));
```

Comment:
```
return orders.filter(order => !hideOrders.some(hideOrderId => order.tokenId.equals(hideOrderId)));
```
NS.TokenId is BigNumber, BigNumber is Object and comparing with `===` won't work.